lms/djangoapps/grades/management/commands/compute_grades.py

```diff
@@ -71,8 +72,28 @@ def add_arguments(self, parser):
             default=0,
             type=int,
         )
+        parser.add_argument(
+            '--estimate_first_attempted',
+            help='Use score data to estimate first_attempted timestamp. Values: (true|false|none)',
+            default='none',
+            choices={'true', 'false', 'none', 'True', 'False', 'None'}
```

Comment:
@iloveagent57 No.  The default behavior is dependent on the global setting of the waffle flag in the database.

@nasthagiri  Because that only lets the caller override the waffle switch in one direction.  If the switch is globally turned on, and the caller wants to run it without estimation, store_true doesn't allow that.  Alternatively, if not passing a store_true value stores False, it doesn't allow the caller to respect the existing waffle flag.

That is to say, I need three values, not two.  If you're sure there's no use case for being able override the waffle flag once it's turned on in the database, I can make the false value mean "use the default," and the true value mean "unconditionally set the waffle switch."